src/components/icon/icon.svelte

```diff
@@ -1,25 +1,34 @@
 <script context="module" lang="ts">
-  let iconBasePath = '/icons'
-  export const setIconBasePath = (basePath: string) => (iconBasePath = basePath)
+  import { writable } from 'svelte/store'
 
-  const getIconUrl = (name: string) => `${iconBasePath}/${name}.svg`
+  let lastIconBasePath = '/icons'
+  let iconBasePath = writable(lastIconBasePath)
+
+  export const setIconBasePath = (basePath: string) => {
+    lastIconBasePath = basePath
+    iconBasePath.set(basePath)
```

Comment:
There's no particular reason why you _should_ do this, but you could also set this by writing `$iconBasePath = basePath`.

Reply:
I get this error
![image](https://user-images.githubusercontent.com/7678024/235802249-4b4f264e-c1a1-40fd-be2f-aca251f793fb.png)

when I try and do that - I think it's because I'm not actually in the component logic here, I'm in the `module` so Svelte's reactivity stuff doesn't work :/

Reply:
Oh! I guess that makes sense.